initialize.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -e
+
```

Comment:
is it possible to write something welcoming the user? 

"this is the helper script to help you install the OS requirements for IntelOwl" or something similar.